python/graphstorm/gsf.py

```diff
@@ -209,61 +234,88 @@ def create_builtin_node_gnn_model(g, config, train_task):
     """
     return create_builtin_node_model(g, config, train_task)
 
-def create_builtin_node_model(g, config, train_task):
-    """ Create a built-in model for node prediction.
+# pylint: disable=unused-argument
+def create_builtin_node_decoder(g, decoder_input_dim, config, train_task):
+    """ create builtin node decoder according to task config
 
     Parameters
     ----------
     g: DGLGraph
-        The graph used in training and testing
+        The graph data.
```

Comment:
Since this method does not use the `g` argument, should we keep it? 
Use `g` as an input argument is really odd.

Reply:
Let's keep it and replace it with g_metainfo in the future.